src/components/Layouts/SidePanelsLayout.tsx

```diff
@@ -129,7 +129,7 @@ export const SidePanelsLayout: FC<SidePanelsLayoutProps> = ({
           flexBasis={leftWidth}
           breakpoint="sm"
         >
-          {leftPanel}
+          <ScrollBox isscroll={isScrollBox ? 1 : 0}>{leftPanel}</ScrollBox>
```

Comment:
There's been some churn on this line. Is this a change you made or a bad merge?

Reply:
Yes I made this change.  I don't know if the left panel used to scroll and then the scrolling was disabled, but I think we need it to scroll again -- the list of filters sometimes extends far past the bottom of the screen.  Especially since we render MultiSelect filters as a long list of checkboxes -- the city filter in particular becomes a verbose list of every individual city that our contacts are from.  I think we need this panel to scroll, independent of the main view.